MAINTAINERS.md

```diff
@@ -14,6 +14,24 @@ Guidelines for issue reports and pull requests are included in this project’s
   - The issue is a request that is deemed “out of scope” for the project.
   - The issue contains content that violates the [Code of Conduct](CODE_OF_CONDUCT.md).
 
+## Managing Results Data
+
+*Note: A new results reporting system is under development, so the following instructions are subject to change.*
+
+In order to improve throughput and limit the impact of intermittent failure, this project subdivides the task of collecting web-platform-tests results. It does this by using the "chunking" functionality of the WPT CLI, splitting the entire test suite into a number of segments and executing the segments in parallel. Once all segments are complete, the build master consolidates the results, uploads them to a central storage location, and notifies a database server of the new data.
+
+Although the system is designed to detect incomplete results and abort the upload process, unexpected errors may allow such invalid datasets to be reported. In this event, maintainers should take the following actions:
+
+1. Create a ticket in this project's issue tracker to describe the error
+2. Update the database record describing the faulty dataset by modifying the
+   `BrowserName` field according to the following pattern:
+
+       invalid-{{browser name}}-{{ticket reference}}
+
+   For instance, to annotate an invalid data set for the "experimental" version of the Firefox browser, referencing ticket 345 on GitHub.com, the `BrowserName` filed should be renamed to:
+
+       invalid-firefox-experimental-gh-345
```

Comment:
Just a note that I would not have guessed that the "gh-" bit was intended to be part of the name unless reading this example. You're probably in the minority of using this syntax on GitHub, I see "#345" much more than "gh-345" :)